site/content/en/references/kustomize/cmd/localize/_index.md

```diff
@@ -0,0 +1,263 @@
+
+<pre>
+<b>kustomize localize</b> [<ins>target</ins> [<ins>newDir</ins>]] [--scope <ins>scope</ins>]
+</pre>
+
+where
+
+* `target` is the [kustomization directory](https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#kustomization-root) 
+to localize. This value can be a local path or a [remote root](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md). 
+The default value is the current working directory.
+* `newDir` is the destination of the "localized" copy that the command
+will create. The destination must reside in an existing directory.
+The default destination is a directory in the current working directory named:
+  * `localized-{target}` for local `target`
+  * `localized-{target}-{ref}`<sup>[[ref]](#notes)</sup> for remote `target`
+* `scope` is the bounding directory. The command can only copy files inside
+this directory. The default is `target`. This flag cannot be specified for
+remote `target`, as its value is implicitly the repo containing `target`.
+
+### Fields
+
+The command "localizes", copying or downloading, files under 
+the following kustomization fields<sup>[[resource]](#notes)</sup>:
+
+* `resources`
+* `bases`
+* `components`
+* `openapi.path`
+* `configurations`
+* `crds`
+* `configMapGenerator`<sup>[[gen]](#notes)</sup>
+* `secretGenerator`<sup>[[gen]](#notes)</sup>
+* `helmChartInflationGenerator`<sup>[[helm]](#notes)</sup>
+* `helmCharts`<sup>[[helm]](#notes)</sup>
+* `helmGlobals`<sup>[[helm]](#notes)</sup>
+* `patches`
+* `patchesJson6902`
+* `patchesStrategicMerge`
+* `replacements`
+
+In addition to localizing files<sup>[[plugin]](#notes)</sup> under the following 
+plugin fields:
+
+* `generators`
+* `transformers`
+* `validators`
+
+the command localizes files referenced by the following plugins<sup>[[resource]](#notes)</sup>,
+which have a built-in kustomization field counterpart:
+
+* `ConfigMapGenerator`<sup>[[gen]](#notes)</sup>
+* `SecretGenerator`<sup>[[gen]](#notes)</sup>
+* `HelmChartInflationGenerator`<sup>[[helm]](#notes)</sup>
+* `PatchTransformer`
+* `PatchJson6902Transformer`
+* `PatchStrategicMergeTransformer`
+* `ReplacementTransformer`
+
+### Structure
+
+The localized destination directory is a copy<sup>[[absolute]](#notes), [[symlink]](#notes)</sup> 
+of `scope`, excluding files that `target` does not reference and 
+with the addition of downloaded remote content.
+
+Downloaded files are copied to a directory named `localized-files` located in
+the same directory as the referencing kustomization. Inside `localized-files`,
+the content of remote 
+* roots are written to path<sup>[[localized root]](#notes)</sup>:
+
+  <pre>
+  <ins>host</ins> / <ins>path/to/repo</ins> / <ins>ref</ins> / <ins>path/to/file/in/repo</ins>
+  </pre>
+
+* files are written to the following path<sup>[[localized file]](#notes)</sup> 
+constructed from its url components:
+
+  <pre>
+  <ins>host</ins> / <ins>path</ins>
+  </pre>
+
+### Example
```

Comment:
This the example below real? i think we shall provide a real example in the kustomize repo and point it there. This is how other kustomize features give examples. 

https://github.com/kubernetes-sigs/kustomize/tree/master/examples

Reply:
Do you mean like this: https://github.com/kubernetes-sigs/cli-experimental/pull/313/files#diff-3de085d06ca2f9e7f7dbd1c0c3c1f86786e2555ac850003aae4ab3875249b8d5R84-R86